pkg/build/builder/sti.go

```diff
@@ -231,12 +265,17 @@ func (s *S2IBuilder) Build() error {
 			}
 			return errors.New(msg)
 		}
-		glog.Infof("Successfully pushed %s", tag)
+		glog.Infof("Successfully pushed %s", pushTag)
 		glog.Flush()
 	}
 	return nil
 }
 
+func (s *S2IBuilder) runPostCommitHook(imageID string) error {
+	glog.Infof("Running post commit hook with image %s ...", imageID)
```

Comment:
also not sure the point of this function.  whoever is calling it should just call runScriptInContainer directly.  (and then it should do the glog.Info)